yarn.lock

```diff
@@ -1204,6 +1204,14 @@
     core-js-pure "^3.0.0"
     regenerator-runtime "^0.13.4"
 
+"@babel/runtime-corejs3@^7.11.2":
```

Comment:
odd that yarn picked up a new one, when there was already one that satisfied the condition...

Reply:
Yeah, but as far as I understand, `yarn install` seems to perform deduplication, so I guess it's the best we can do.